cli/ballerina-cli/src/main/resources/cli-help/ballerina-help.help

```diff
@@ -1,5 +1,5 @@
 NAME
-       The bal build tool
+       The ballerina update tool
```

Comment:
Is this correct? Shouldn't this be `ballerina build` tool?

Reply:
Noted. This is on the discussion. Not Finalized yet. https://github.com/ballerina-platform/ballerina-update-tool/issues/71. Shall we check with others as well, please?

Reply:
Its not just update tool or build tool, Lets change it to `The Ballerina tool`

Reply:
Ballerina with the uppercase B

Reply:
Noted.